worker/codalabworker/docker_client.py

```diff
@@ -436,7 +438,9 @@ def start_container(self, bundle_path, uuid, command, docker_image,
             'Entrypoint': [""], # unset entry point regardless of image
             'HostConfig': {
                 'Binds': volume_bindings,
+                'NetworkMode': network_name,
```

Comment:
Why is this part of this PR?

Reply:
Just a small re-arrangement, moving the same functionality from line 450 to here.